ansible/Ansible.common.mk

```diff
@@ -17,16 +17,11 @@ $(ROLES_PATH) $(COLLECTIONS_PATH):
 	@mkdir -p $@
 
 # Installs dependencies into "collections" and "roles" folders
-.PHONY: ansible/dependencies
-ansible/dependencies: $(ROLES_PATH) $(COLLECTIONS_PATH)
+.PHONY: dependencies
+dependencies: $(ROLES_PATH) $(COLLECTIONS_PATH)
 	ansible-galaxy role install -r $(REQUIREMENTS_FILE) -p $(ROLES_PATH)
 	ansible-galaxy collection install -r $(REQUIREMENTS_FILE) -p $(COLLECTIONS_PATH)
 
-.PHONY: ansible/prepare-inventory
-ansible/prepare-inventory:
-	@sed "s/PR_NUMBER/${PR_NUMBER}/g" $(ANSIBLE_INVENTORY_TEMPLATE) > $(ANSIBLE_INVENTORY)
-
-# Removes "collections" and "roles" folders
-.PHONY: ansible/clean
-ansible/clean:
-	rm -rf $(ROLES_PATH) $(COLLECTIONS_PATH)
```

Comment:
Can't we keep the `clean` target here as well?

Reply:
I try to state that clearly here https://github.com/newrelic/fluent-bit-package/pull/110/files#diff-35ea617bfbf0780bfbc554348314c5b11ba478a4d98fd92d33d45cdd54bf326aR18-R21

Only targets referenced from top-level targets can be put in _common_ files. The root makefile does not follow all called targets within makefiles to figure out how to populate them. 

I think it is a _small_(tiny) drawback. It is overall simple to follow.

wdty?